src/cmds/scripts/win_postinstall.py

```diff
@@ -41,7 +41,7 @@ def ValidateCredentials(username, password):
                 print "Post install process started..."
                 print "User Validation Success"
     else:
-        print "Usage error: \n post_install.py -u <username> -p <password>"
+        print "Usage error: \n win_post_install.py -u <username> -p <password>"
```

Comment:
Please use the `__file__` directive so that we do not need to update the usage message every time the file name changes.